src/Magazine/Magazine.Domain/AggregatesModel/Magazine/Magazine.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Magazine.Domain.AggregatesModel;
+public class Magazine
+{
+    public Guid Id { get; set; }
```

Comment:
`public set`, т.е. мы можем изменять его из вне. Может лучше:
```suggestion
    public Guid Id { get; init; }
```
Может стоит рассмотреть возможность сделать `set` `init`?